tox.ini

```diff
@@ -4,7 +4,10 @@ envlist = py38, py39, py310, py311, lint-incr
 isolated_build = true
 
 [testenv]
-usedevelop = True
+# We pretend that we're not actually installing the package, because we need tox to let us have two
+# packages ('qiskit' and 'qiskit-terra') under test at the same time.  For that, we have to stuff
+# them into 'deps'.
+skip_install = true
```

Comment:
I"m a bit concerned about this, I seem to remember `skip_install` causing weird behavior in tox >4.0.0, I think with regards to tracking source updates or something, I don't fully remember but I know it caused issues for me on the upgrade to 4.0.0.

Reply:
I'm open to other ideas here, this is just the only thing I could think of to trick `tox` into letting us put two local packages under test at the same time.  I'm moving ever further into the "let's ditch tox" camp.